Bio/SeqUtils/NucUtils.py

```diff
@@ -0,0 +1,191 @@
+# Copyright 2018 Aleksandra Jarmolinska.  All rights reserved.
```

Comment:
This should probably acknowledge being based on ``Bio/SeqUtils/___init__.py`` and if used Leighton Pritchard's code in GenomeDiagram's ``Utilities.py``.

Reply:
I wasn't sure about the proper form so added authors of __init__.py as "Based on code by ...". Didn't use GenomeDiagram